docs/source/install/monitor_without_docker.rst

```diff
@@ -1,16 +1,32 @@
+================================================
 Deploying Scylla Monitoring Stack Without Docker
 ================================================
-
+Introduction
+------------
 The following instructions will help to deploy `Scylla Monitoring Stack <monitoring_stack>`_ in cases where you can not use the recommended Docker version.
 
 Please note, Scylla recommends you use the Docker version as it will provide you with most updated, current Scylla Monitoring system.
 
+Scylla Monitoring uses the following components:
+
+* Alertmanager
+* Grafana Loki
+* Prometheus
+* Grafana
+
+The common scenario for users who use their own standalone installation, is they already have such a server and would like to consolidate.
+We assume that that you already have Prometheus and Grafana running but we will add minimal installation instruction for all componenents.
```

Comment:
```suggestion
```

Reply:
I want to emphesize that this is not instead of reading the actual installation documentation.